services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/config.py

```diff
@@ -0,0 +1,105 @@
+# this is a proxy for the global configuration and will be removed in the future
+# making it easy to refactor
+
+from aiohttp.web import Application
+from pydantic import BaseSettings, Field, PositiveInt
+
+
+class DynamicSidecarSettings(BaseSettings):
+    # dynamic_sidecar integration
+    monitor_interval_seconds: float = Field(
+        5.0, description="interval at which the monitor cycle is repeated"
+    )
+
+    max_status_api_duration: float = Field(
+        1.0,
+        description=(
+            "when requesting the status of a service this is the "
+            "maximum amount of time the request can last"
+        ),
+    )
+
+    traefik_simcore_zone: str = Field(
+        "internal_simcore_stack",
+        description="used by Traefik to properly handle forwarding of requests",
+        env="TRAEFIK_SIMCORE_ZONE",
+    )
+
+    swarm_stack_name: str = Field(
+        "undefined-please-check",
+        description="used to filter out docker components from other deployments running on same swarm",
+        env="SWARM_STACK_NAME",
+    )
+
+    dev_simcore_dynamic_sidecar_path: str = Field(
+        None,
+        description="optional, only used for development, mounts the source of the dynamic-sidecar",
+        env="DEV_SIMCORE_DYNAMIC_SIDECAR_PATH",
+    )
+
+    image: str = Field(
```

Comment:
this could also be validated. I'm sure you can find plenty of regexp for docker images

Reply:
I would actually skip validating this field. Not that obvious how to do it properly.

Reply:
look into models-library we have a regexp for our own services. I guess you can go along the same lines.